triangle/triangle.py

```diff
@@ -0,0 +1,79 @@
+import sys
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 now it works if in string less then 6 coordinates 
 Mistakes: 
 Where have you found such homework(now)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(it), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(works)? It is not from ['Added', 'Deleted'].